apps/storybook/.storybook/image-storyshots.__test__.js

```diff
@@ -1,16 +1,12 @@
 import initStoryshots from '@storybook/addon-storyshots';
 import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';
 import path from 'path';
-
-initStoryshots({
-  suite: 'JSON',
-});
-
 // In the build command we export a variable which helps us detect a web env.
 // visual regression testing only works in the browser
 if (process.env.BABEL_ENV === 'test:web' && !process.env.IS_EXPO_CI) {
   // Locally it's better to start the server and then run the tests.
   // In CI we want to build a static bundle and test against that.
+  // const storybookUrl = 'file://' + path.resolve(__dirname, '..', 'web-build')
```

Comment:
remove this

Reply:
It works locally but breaks if used in CI. WIP